auslib/test/blobs/test_superblob.py

```diff
@@ -19,7 +21,39 @@ def setUp(self):
     ]
 }
 """)
+        self.superblob_addon = SuperBlob()
+        self.superblob_addon.loadJSON("""
+{
+    "name": "fake",
+    "schema_version": 6000,
```

Comment:
Please update this schema number to be correct - there is no blob with schema 6000!